src/contrib/mlir/compiler.hpp

```diff
@@ -98,13 +98,18 @@ namespace ngraph
 
                 void build_ng_dialect();
 
-                template <typename OP>
+                template <typename Op>
                 static mlir::Value* create_op(MLIRCompiler& compiler, const ngraph::Node* ng_node)
                 {
                     throw std::runtime_error("Unimplemented op '" + ng_node->description() +
                                              "' in MLIR Compiler");
                 }
 
+                // Generic op lowerer to ng dialect.
+                // Simply maps ngraph tensors to values and generate an OP. No op-specific logic.
+                template <typename OP>
+                mlir::Value* create_generic_op(const ngraph::Node* ng_node);
+
                 template <typename UnaryOp>
                 mlir::Value* create_unary_op(const ngraph::Node* ng_node);
```

Comment:
I think we should remove this prototype now, along with that for `create_binary_op` etc.